index.html

```diff
@@ -176,25 +171,50 @@ <h1 class="about-me-title">About me</h1>
             </ul>
           </div>
         </div>
-
+        
         <div class="card skills-card">
-          <img src="assets/images/skills-icon.png" alt="icon">
+          <div class="card-img">
+            <img src="assets/images/skills-icon.png" alt="icon">
+          </div>
           <div class="card-details">
             <p>Skills</p> 
             <ul class="card-tags"> 
               <li>Codekit</li>
               <li>GitHub</li>
               <li>Codepen</li>
               <li>Gitlab</li>
-              <li>Ternimal</li>
+              <li>Terminal</li>
             </ul>
           </div>
         </div>
-        
       </div>
-
+      
+    </section>
+    <section class="form">
+      <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
```

Comment:
- [ ] Please change `<h1>` tag to another heading tag example `<h2>` , `<h3>` , `<h4>`. Your website should have only one `<h1>` tag and you are already using that in your hero section.